src/indexeddb.py

```diff
@@ -0,0 +1,190 @@
+            "description": "The browser type",
+            "items": [ "chromium", "firefox", "safari" ],
+            "type": "select",  # Types supported: text, textarea, checkbox
+            "required": True,
+        },
+        {
+            "name": "output_format",
+            "label": "Select output format",
+            "description": "The output format",
+            "items": [ "JSON", "JSONL", "REPR" ],
+            "type": "select",  # Types supported: text, textarea, checkbox
+            "required": True,
+        }
+    ],
+}
+
+INTERVAL_SECONDS = 2
+
+
+@celery.task(bind=True, name=TASK_NAME, metadata=TASK_METADATA)
+def command(
+    self,
+    pipe_result: str | None = None,
+    input_files: list | None = None,
+    output_path: str | None = None,
+    workflow_id: str | None = None,
+    task_config: dict | None = None,
+) -> str:
+    """Run dfindexeddb on input files.
+
+    Args:
+        pipe_result: Base64-encoded result from the previous Celery task, if any.
+        input_files: List of input file dictionaries (unused if pipe_result exists).
+        output_path: Path to the output directory.
+        workflow_id: ID of the workflow.
+        task_config: User configuration for the task.
+
+    Returns:
+        Base64-encoded dictionary containing task results.
+    """
+    input_files = get_input_files(pipe_result, input_files or [])
+    output_files = []
+    base_command = "dfindexeddb"
+
+    if not task_config:
+        return create_task_result(
+            output_files=output_files,
+            workflow_id=workflow_id,
+            command=base_command,
+            meta={},
+        )
+
+    # parse task configuration
+    output_format = task_config.get("output_format", []).lower()
+    output_config = definitions.OUTPUT_TYPES_EXTENSIONS[output_format]
+    output_extension = output_config["extension"]
+    browser_type = task_config.get("browser_type", "")
+
+    # Create temporary directory and hard link files for processing
+    temp_dir = os.path.join(output_path, uuid.uuid4().hex)
+    os.mkdir(temp_dir)
+
+    input_files_temp = []
+    for input_file in input_files or []:
+        display_name = input_file.get("display_name")
+        temp_file = f"{temp_dir}/{display_name}"
+        shutil.copy(input_file.get("path"), temp_file)
+        input_files_temp.append((input_file, temp_file))
+
+    for input_file, temp_file in input_files_temp:
+        display_name = input_file.get("display_name")
+        original_path = input_file.get("path")
+        source_file_id = input_file.get("id")
+        data_type = f"openrelik:dfindexeddb:{browser_type}:{output_format}"
+        if browser_type == "chromium":
+            for subcommand, value in definitions.CHROMIUM_FILE_REGEX.items():
+                if re.search(value, display_name):
+                    break
+            else:
+                print(f"Unsupported {browser_type} file type for {display_name}.")
```

Comment:
Do we need to print here? If yes, let's move this to a logger instead. We can do this in a followup PR though.